packages/blade/src/components/Box/__tests__/baseBoxStyles.native.test.ts

```diff
@@ -18,7 +18,6 @@ describe('getResponsiveValue', () => {
 
   it('should correctly handle falsy values', () => {
     expect(getResponsiveValue(undefined)).toBe(undefined);
-    // @ts-expect-error: intentional null to check the falsy values
```

Comment:
null is now a valid value since React Native added `DimensionValue` which has `null` in some style props like flexBasis & opacity